site/utils/apiUtils/index.ts

```diff
@@ -99,7 +99,7 @@ export const getPageState = <T>(errorCookie: string, schemaObject: z.ZodType<T>,
         return { ...(JSON.parse(errorCookie) as PageState<Partial<T>>), disruptionId: disruptionId || "" };
     }
 
-    if (disruptionId) {
+    if (disruptionId || !!data) {
```

Comment:
Original comment: should this be &&? what if there is no data or disruptionId do we want to throw an error?

I used an or expression because when I use this function for the edit-user page I don't have a disruptionId to pass through. Looking at the usages there is at least one occasion where we pass through a disruptionId without data so I think this logic is ok and we do not need to throw an error.